lib/farmoney/money.rb

```diff
@@ -36,6 +36,25 @@ def /(other)
     def *(other)
       Money.new(pence * other)
     end
+
+    def lt(other)
+      pence < other.pence
+    end
+
+    def gt(other)
+      pence > other.pence
+    end
+
+    def eq(other)
+      pence == other.pence
+    end
+
+    def lteq(other)
+      lt(other) || eq(other)
+    end
+
+    def gteq(other)
+      gt(other) || eq(other)
+    end
```

Comment:
Its better to use `> < =` etc. instead of these names.

You can define a `<=>` method, and `include Comparable`: https://ruby-doc.org/core-2.5.0/Comparable.html